fiftyone/utils/multiprocessing.py

```
@@ -0,0 +1,294 @@
+            range(num_shards),
+            itertools.repeat(num_shards),
+            slices,
+        )
+    )
+
+    num_workers = min(num_workers, num_shards)
+
+    return batches, num_workers
+
+
+def _init_worker(dataset_name, view_stages, m, s, r, p, l):
```

Ruff (0.8.2)

Ambiguous variable name: `l`

(E741)